3.5/Backup_Restore.md

```diff
@@ -0,0 +1,299 @@
+# Back up and Restire CP4D Volume
+
+## CP4D Volume Backup and Restore Using cpdbr 2.0
+
+You need to carry out this pre-req step befor upgrading CP4D services. The following steps describes how to perform off-line backup and restore of data volumes using the CPD backup utility cpdbr.  To provide data consistency, cpdbr quiesces services before performing a backup.
+
+The cpdbr 2.0 CLI released for CPD 3.5 is encapsulated by a top-level CLI called cpd-cli.  cpd-cli invokes cpdbr internally as an executable plugin to perform backup and restore.  The equivalent of running cpdbr <command> is cpd-cli backup-restore <command>.
+
+### Installing cpdbr
+
+1.	Install cpdbr 2.0
+
+  cpdbr consists of a CLI utility and docker image.
+
+  Download the “cpd-cli” CLI
+
+  ```bash
+  wget https://github.com/IBM/cpd-cli/releases/download/v3.5.0/cpd-cli-linux-EE-3.5.1.tgz
+  tar xvf cpd-cli-linux-EE-3.5.1.tgz
+  ```
+
+Check cpdbr version
+
+  ```bash
+  ./cpd-cli backup-restore version
+  ```
+
+  ```bash
+  backup-restore
+          Version: 2.0.0
+          Build Date: 2020-10-28T23:00:47
+          Build Number: 730
+          CPD Release Version: 3.5.1
+  ```
+
+2. Get the `BUILD_NUM` for cpdbr image
+
+  ```bash
+  BUILD_NUM=`./cpd-cli backup-restore version | grep "Build Number" |cut -d : -f 2 | xargs`
+  ```
+
+3. Save the image using docker/podman commands
```

Comment:
Is "podman" required?  This is an extra utility that isn't automatically installed for OpenShift.  I have questions about this.  Since you are showing docker commands, I would remove "podman"

Reply:
You can do either podman or docker. I did not podman installed so I used docker but any user with podman should be able to run the same commands so I provided the options as podman is used everywhere in the docs